Library/Formula/moc.rb

```diff
@@ -37,8 +34,12 @@ class Moc < Formula
   depends_on "libmagic" => :optional
   depends_on "homebrew/dupes/ncurses" => :optional
 
+  # Backport r2779: Adapt to FFmpeg/LibAV's audioconvert.h rename (commit 5980f5dd).
+  # Necessary for building against FFmpeg 3.0. See http://moc.daper.net/node/1496.
+  patch :DATA
```

Comment:
Since this is a fairly significant patch in size let's get it submitted to https://github.com/Homebrew/patches and use that URL instead.